H_ecto_models.md

```diff
@@ -621,11 +624,11 @@ end
 ```
 Now, when `Repo.update` is called for the `Video` model, the `approved_at` field will automatically be reset to nil. This should keep us safe from the ravages of an often hostile modern Internet!
 
-`Ecto.Model.Callbacks` actually ships with with a lot more than the `before_update` callback, in addition, it comes with:
+`Ecto.Model.Callbacks` actually ships with a lot more than the `before_update` callback, in addition, it comes with:
 
 - `before_delete`: called before the adapter deletes our record from the database; if perhaps we have a foreign_key constraint from another table, we would use this hook to clean it up.
 - `after_delete`: called after the adapter deletes our record. If we wished to archive our record, we'd do it in this hook.
-- `before_update`: called before the adapter updates our record. We used it above to reset the likes counter.
+- `before_update`: called before the adapter updates our record. We used it above to "null out" a video's `approved_at` field.
 - `after_update`: called after the adapter updates our record. If we wanted to update our user that the model has changed, now would do that here.
 - `before_insert`: called before the adapter first creates a new record. We would use it in cases when we wanted to infer default values to certain fields from the partial changeset we receive from the user.
 - `after_insert`: called after the adapter creates our record. We could use this to notify the user a new has been created under that user's account.
```

Comment:
Maybe "We could use this to notify the user a new **video** has been created under that user's account."?